src/main/utils/PlaybackAPI.js

```diff
@@ -35,6 +42,10 @@ class PlaybackAPI {
         album: null,
         albumArt: null,
       },
+      rating: {
+        liked: false,
```

Comment:
I didn't even know you could have more than like/dislike in GM.
I'll look into it.